### PR TITLE
[#IC-341] add message avro model

### PR DESCRIPTION
**Pull request by gquadrati**
#### List of Changes

- Add new avro model for messages (and their contents)

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by fabriziopapi — COMMENTED** [review]
Are we sure to not send the hased fiscal code?

**Comment by gquadrati** [comment]
> Are we sure to not send the hased fiscal code?

With hashed fiscal code messages can be aggregated, and some kind of "guessing strategy" can be performed over them

**Comment by gquadrati** [comment]
> Are we sure to not send the hased fiscal code?

With hashed fiscal code messages can be aggregated, and some kind of "guessing strategy" can be performed over them

**Comment by fabriziopapi** [comment]
> > Are we sure to not send the hased fiscal code?
> 
> With hashed fiscal code messages can be aggregated, and some kind of "guessing strategy" can be performed over them

True, but we'll lose a lot of possible aggregations.
Let's hope we'll not have to re-process messages soon.

**Review by fabriziopapi — APPROVED** [review]

File: avro/message.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      "logicalType": "time-millis",
+      "default": 0
+    },
+    {
+      "name": "isPending",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "content",
```
Comment (fabriziopapi) [inline avro/message.json:51]: pdnd asked us to provide data flattened (when possible). Maybe we can flatten content and payment data fields directly on the root.

**Pull request merged**